client/components/board/boardFactory.js

```diff
@@ -3,27 +3,48 @@ angular.module('OpenEXP')
         const OpenBCIBoard = require('openbci-sdk');
         const ourBoard = new OpenBCIBoard.OpenBCIBoard();
 
-        var findPorts = () => {
+        // wrote a method in the openbci-sdk to return an array of ports and
+        // wrapping it in a promise here
+        var listDevices = () => {
```

Comment:
more comment than question: I guess that this will need to be more general in the future and not rely on the sdk for listing the ports